source/data_classify.py

```diff
@@ -0,0 +1,59 @@
+"""This script takes in a preprocessed dataset and trains a classifier,
+it finds the accuracy score and bias factor
+"""
+
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+import sklearn as sk
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import StandardScaler
+from sklearn.compose import ColumnTransformer
+from sklearn.metrics import accuracy_score, roc_auc_score
+from sklearn.utils.class_weight import compute_class_weight
+from sklearn.preprocessing import OneHotEncoder
+
+from data_operations import nn_classifier
+from data_operations import p_rule
+
+data_input = pd.read_csv('../data/processed_adult.csv',low_memory=False)
```

Comment:
It's not necessary to do this now, but allowing this to be a command line argument would be a nice feature.